docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -0,0 +1,372 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:eip20: https://eips.ethereum.org/EIPS/eip-20[EIP-20]
+:erc20-guide: xref:erc20.adoc[ERC20 guide]
+
+
+= ERC20
+
+Reference of interfaces and utilities related to ERC20 contracts.
+
+TIP: For an overview of ERC20, read our {erc20-guide}.
+
+== Core
+
+[.contract]
+[[IERC20]]
+=== `++IERC20++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc20/interface.cairo#L6-L19[{github-icon},role=heading-link]
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::token::erc20::interface::IERC20;
+```
```

Comment:
do we really prefer the light theme here? cc/ @ericnordelo @andrew-fleming

Reply:
Good question. Actually, I think I lean toward the dark version because makes it easier to note the separation between different sections. Not strong opinion though, I'm ok with light to keep consistency with the solidity doc site.

<img width="768" alt="Screenshot 2023-09-12 at 14 16 46" src="https://github.com/OpenZeppelin/cairo-contracts/assets/26070224/924ce4ec-85ba-488c-bec8-f8f0e2d0352f">

Reply:
+1 to the dark theme